src/content/docs/en/editor-setup.mdx

```diff
@@ -4,8 +4,9 @@ description: Set up your editor to build with Astro.
 i18nReady: true
 ---
 
-import Badge from '~/components/Badge.astro';
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import { Badge } from "@astrojs/starlight/components"
+import "~/styles/custom.css" // import custom.css for customize starlight components
```

Comment:
Import `custom.css` instead of Starlight `customCss` option.

Because `customCss` option will bundle to a external style link tag. It can't override Starlight component's scoped style.

But, I only import it in `/en/editor-setup.mdx`, it works for all pages.